### PR TITLE
[Validator] Add Dutch translations

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no     
| Deprecations? | no
| Tests pass?   | yes    
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

```
| Locale: nl      | Translated: 90/90
```

### CODE REVIEW

**Review by TimoBakx — APPROVED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 1]
```diff
@@ -330,6 +330,38 @@
...
+                <source>This value should be negative.</source>
+                <target>Deze waarde moet negatief zijn.</target>
+            </trans-unit>
+            <trans-unit id="91">
+                <source>This value should be either negative or zero.</source>
+                <target>Deze waarde moet negatief of gelijk aan nul zijn.</target>
+            </trans-unit>
+            <trans-unit id="92">
+                <source>This value is not a valid timezone.</source>
+                <target>Deze waarde is geen geldige tijdszone.</target>
```
Comment (linaori) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Should be `tijdzone` according to google

Comment (ro0NL) [reply src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: fixed.

**Review by linaori — APPROVED** [review]
The translations seem to be correct :+1:

**Review by javiereguiluz — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request merged**